flowzone.yml

```diff
@@ -2183,6 +2185,18 @@ jobs:
             echo  "::error::Failed to get branch protection rules ${message} ${result} "
             exit 1
           fi
+
+      # disable auto-merge on any existing PRs if the required reviewer count is decreasing
+      - name: Disable PR auto-merge
+        id: disable_auto_merge
+        if: |
+          fromJSON(steps.get_protection_rules.outputs.result).required_pull_request_reviews.required_approving_review_count < fromJSON(inputs.inputs.required_approving_review_count)
+        run: |
+          for pr in $(gh pr list --state=open --draft=false --json=number | jq -r '.[].number')
+          do
+            gh pr merge ${pr} --merge --disable-auto || true
```

Comment:
I'm confused by what this command does. The [github docs](https://cli.github.com/manual/gh_pr_merge) for the cli says `gh pr merge $` will "Merge a pull request on GitHub.".

`--merge` described as "Merge the commits with the base branch"
and `--disable-auto` "Disable auto-merge for this pull request".

I'm unable to tell if this is merging the PR or telling the PR to not auto merge once it is approved.